src/pages/preList/detail/index.tsx

```
@@ -10,18 +10,22 @@ const PreviousDetail = () => {
               userInfo={item.date}
-              time={`${item.start_time} - ${item.end_time}`}
+              time={
+                item.end_time
+                  ? `${item.start_time} ~ ${item.end_time}`
+                  : `${item.start_time} ~`
+              }
               reason={item.reason}
               selected={false}
               onClick={() => {}}
             />
           ))
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)